docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -0,0 +1,207 @@
+trim down, is that Galleon only install the needed content (JBOSS modules, scripts).
+
+To install a server only configured with jaxrs and cdi functionalities call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --layers=cdi,jaxrs
+----
+
+Once installed, the directory _my-wildfly-server_ contains all that is expected to deploy an application
+that depends on _jaxrs_ and _cdi_. 
+
+NB:
+
+* Tools (jboss-cli, add-user, ...) are not installed when installing WildFly using layers. 
+Only server launcher scripts are installed into _bin_ directory.
+
+=== Defined Galleon layers
+
+==== Note on layers and feature-pack dependencies
+
+When installing a feature-pack, its own defined layers as well as the layers present in its dependencies 
+can be combined to build a custom configuration. This means that core and servlet defined layers can be combined with 
+WildFly defined layers to assemble a custom configuration. <<galleon-cli-recipes>> contains examples of such combination of layers.
+
+==== WildFly layers
+
+Basic layers:
+
+* _cdi_: Support for cdi.
+* _jaxrs_: Support for jaxrs.
+* _jms-activemq_: Support for connection to remote jms broker. 
+* _jpa_: Support for jpa (latest WildFly supported hibernate release).
+* _micro-profile_: Support for microprofile (config, health, metrics, open-tracing).
+* _resource-adapters_: Support for deployment of JCA adapters.
+* _h2-database_: Support for h2 database.
+
+Aggregation layers:
+
+* _cloud-profile_, an aggregation of the basic layers (except _h2-database_).
+
+NB: These layers depend at least on the _web-server_ layer defined in WildFly servlet 
+feature-pack (on which WildFly feature-pack depends). This means that when creating an 
+installation with these layers, the installed WildFly server is also configured to be a servlet container. 
+
+==== WildFly servlet layers
+
+Basic layers:
+
+* _ee_: Support for common functionalities in the Java EE platform.
+* _legacy-security_: Support for legacy web security. 
+* _naming_: Support for JNDI.
+* _undertow_: Support for undertow HTTP server.
+* _undertow-load-balancer_: Support for undertow configured as a load balancer.
+* _vault_: Support for vault (Picketbox).
+
+Servlet support layer:
+
+* _web-server_: A servlet container composed of _ee_, _naming_, _deployment-scanner_ 
+(from WildFly core feature-pack) and _undertow_ configured as a servlet container.
+
+==== WildFly core layers
+
+Basic layers:
+
+* _base-server_: Empty runnable server.
+* _core-management_: Support for server management services.
+* _deployment-scanner_: Support for deployment directory scanning.
+* _discovery_: Support for discovery.
+* _elytron_: Support for elytron security.
+* _io_: Support for XNIO workers and buffer pools.
+* _jmx_: Support for registration of Management Model MBeans.
+* _jmx-remoting_: Support for registration of Management Model MBeans and JMX remoting connector.
+* _logging_: Support for logging.
```

Comment:
s/logging/the logging subsystem/g

Or some other suggestion. Basic point is there is logging support without the subsystem.